pallets/parachain-staking/src/lib.rs

```diff
@@ -363,7 +363,7 @@ pub mod pallet {
 		> Default for RoundInfo<B>
 	{
 		fn default() -> RoundInfo<B> {
-			RoundInfo::new(1u32, 1u32.into(), 20u32.into())
+			RoundInfo::new(1u32, 1u32.into(), 20u32)
```

Comment:
this change was/is made in https://github.com/PureStake/moonbeam/pull/489

Reply:
Nice. I'll revert it here.

Reply:
447ca8f

Reply:
Oh yeah, you've made some other clippy-friendly changes there too.